envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/synchronization/RemoteServicesTest.kt

```diff
@@ -37,37 +51,24 @@ class RemoteServicesTest {
     }
 
     @Test
-    fun `should timeout long running http call and try again`() {
```

Comment:
Removed it, because behaviour has changed -> when there is timeout and no state in cache, we are sending event with empty list of services and it is checked in other test cases,